### PR TITLE
Module cleanup in DQM/L1TMonitor

### PR DESCRIPTION
**Pull request by Dr15Jones**
Made the following modifications in L1TRCT and L1TRPCTF
- removed unused member variables
- put globals into anonymous namespace to avoid poluting the global
  namespace
- remove trivial member functions associated with Run and LuminosityBlock
  transitions.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

**Comment by andrius-k** [comment]
+1

File: DQM/L1TMonitor/src/L1TRCT.cc [hunk 1]
```diff
@@ -16,22 +16,23 @@
 
 using namespace edm;
 
-const unsigned int PHIBINS = 18;
-const float PHIMIN = -0.5;
-const float PHIMAX = 17.5;
+namespace {
```
Comment (fabiocos) [inline DQM/L1TMonitor/src/L1TRCT.cc]: @Dr15Jones for my understanding, what is the purpose of the definition of this unnamed namespace? Is it to ensure ot restrict the definition of the constants to the scope of this class (through this file as translation unit)?

**Comment by fabiocos** [comment]
+1

**Pull request merged**